src/main/webapp/success_request.jsp

```diff
@@ -0,0 +1,12 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<html>
+<head>
+    <title>Title</title>
+</head>
+<body>
+<h1>Success!</h1>
+<p>Name: <%= request.getAttribute("name")%></p>
```

Comment:
Avoid usage of scriplets

Reply:
Fixed web.xml, now all works without scriptlets.